src/pages/Graph/GraphToolbar/GraphFind.tsx

```diff
@@ -698,6 +702,19 @@ export class GraphFind extends React.Component<GraphFindProps, GraphFindState> {
       case 'op':
       case 'operation':
         return { target: 'node', selector: `[${CyNode.aggregateValue} ${op} "${val}"]` };
+      case 'rank': {
+        if (!this.props.rank) {
+          AlertUtils.addSuccess('Enabling "rank" display option for graph find/hide expression');
+          this.props.toggleRank();
+        }
```

Comment:
@jshaughn one thing that @mattmahoneyrh brought up during his testing is that when `rank = X` is in the find/hide box, the user is unable to deselect the "rank" display option. This is because of the autotoggle here. Other display options with the autotoggle like response time work similarly. Is this intended or is it a bug?

Reply:
![display_options_deselect](https://user-images.githubusercontent.com/6226732/141186409-60b4a5be-b06b-4be2-8098-43ae875ebd45.gif)

Reply:
It's correct.  It actually gets deselected and re-enabled quickly.  You can't have the find/hide expression and not have it enabled.  The same behavior can be seen with mtls and the security option, for example.

Reply:
if you look closely you may notice the notification count go up.  anyway, it's correct behavior. ship it.

Reply:
@mattmahoneyrh ^^